katsdpingest/sigproc.py

```diff
@@ -255,15 +257,17 @@ def generate(block, vtx, vty):
             vtx=[1, 2, 3, 4],
             vty=[1, 2, 3, 4])
 
-    def instantiate(self, command_queue, channels, baselines, masks):
-        return PrepareFlags(self, command_queue, channels, baselines, masks)
+    def instantiate(self, command_queue, channels, baselines, masks, zero_flag):
+        return PrepareFlags(self, command_queue, channels, baselines, masks, zero_flag)
 
 
 class PrepareFlags(accel.Operation):
     """Concrete instance of :class:`PrepareFlagsTemplate`
 
     .. rubric:: Slots
 
+    **vis** : baselines × channels, complex64
```

Comment:
Out of curiosity, why is the visibilities transposed relative to the flags?

Reply:
They just happen to be convenient to get in that orientation. But actually the visibilities going into the flagger have the same orientation as the flags (the visibilities get transposed), so I think it should be possible to make things consistent here too, which would be less confusing. Let me do that.